BUILDING.md

```diff
@@ -181,11 +179,10 @@ $ ./configure
 $ make -j4
 ```
 
-Running `make` with the `-j4` flag will cause it to run 4 compilation jobs
-concurrently which may significantly reduce build time. The number after `-j`
-can be changed to best suit the number of processor cores on your machine. If
-you run into problems running `make` with concurrency, try running it without
-the `-j4` flag. See the
+The `-j4` option will cause `make` to run 4 simultaneous compilation jobs which
```

Comment:
```Suggestion 
The `-j` option indicates to `make` to run in parallel mode, the number after `-j` should best suit the number of CPU cores on your machine.

If you run into problems running `make` see the [GNU Make Documentation](https://www.gnu.org/software/make/manual/html_node/Parallel.html) for more information.
```

Reply:
I think the suggestion here is less clear than the text that is in the PR. What is the crux of the issue for you? I'm probably just not understanding what you're trying to get at...

Reply:
I mostly want it shorter. Less words about `make` when we can refer to the docs.
Maybe just:
```
The `-j4` option will cause `make` to run 4 simultaneous compilation jobs which
may reduce build time. For more information, or in case you run into problems
see the ...
```

Reply:
Or even:
```md
For more information, or in case you run into problems see the ...
```

Reply:
I went with:

```markdown
The `-j4` option will cause `make` to run 4 simultaneous compilation jobs which
may reduce build time. For more information, see the
[GNU Make Documentation](https://www.gnu.org/software/make/manual/html_node/Parallel.html).
```

Hope that works for you! If not, comment again, of course!